gunicorn/config.py

```diff
@@ -1666,3 +1670,13 @@ class StatsdHost(Setting):
     desc ="""\
     host:port of the statsd server to log to
     """
+
+class SpawnWorkerSleepTime(Setting):
+    name = "spawn_worker_sleep_time"
+    section = "Worker Processes"
+    cli = ['--spawn-worker-sleep-time']
+    validator = validate_float
+    default = 0.1
+    desc = """\
+    Time to sleep between spawning workers.
```

Comment:
nit: "Time in seconds..."  Alternatively, you could specify it in milliseconds and avoid the extra float validation.